requirements.txt

```diff
@@ -1,13 +1,15 @@
 #
-# This file is autogenerated by pip-compile with python 3.10
-# To update, run:
+# This file is autogenerated by pip-compile with Python 3.10
+# by the following command:
 #
 #    pip-compile
 #
 beautifulsoup4==4.11.1
     # via ixbrlparse (setup.py)
 lxml==4.9.1
     # via ixbrlparse (setup.py)
+regex==2022.10.31
+    # via ixbrlparse (setup.py)
```

Comment:
I'm slightly wary of adding another dependency rather than just using the standard library - if there are parts of the new format functions that require this library then that's fine, but if it's just performance I think I'd rather stick with the standard library.